DEVELOPER.md

```diff
@@ -6,33 +6,54 @@ flowchart TD
     ApiServer[Laserfiche Repository]-- HTTP Response -->OkHttp3-- HTTP Response -->deserialization[Custom Deserialization]-- Java Objects -->RepositoryApiClient
     OAuthInterceptor<-- Request/Response for Access Token -->oauth[OAuth Service]
 ```
+
 |Entity|Details|Interface|Class|
 |-|-|-|-|
-|RepositoryApiClient|This is the public interface the developer uses to get data from and set value to the Laserfiche Repository|RepositoryApiClient|RepositoryApiClientImpl|
-|OAuthInterceptor|It uses [lf-api-client-core-java](https://github.com/Laserfiche/lf-api-client-core-java) to get access token from the OAuth service, and converts the token into a Bearer header for each outgoing HTTP request.|okhttp3.interceptor|OAuthInterceptor|
-|[Retrofit2](https://square.github.io/retrofit/)/[OkHttp3](https://square.github.io/okhttp/)|We use Swagger Codegen to generate Retrofit methods to communicate with the Laserfiche Repository. We also have hand written Retrofit methods to implement nextLink related APIs.|Anything under com.laserfiche.repository.api.clients.impl|Retrofit only has interface.|
-|OAuth Service|Laserfiche OAuth service|See [lf-api-client-core-java](https://github.com/Laserfiche/lf-api-client-core-java)|N/A|
-|Custom Deserialization|Ideally we would rely on the auto generated code but it fails to handle anyOf, oneOf, allOf so we have our own code for it.|Anything under com.laserfiche.repository.api.serialization|No interface|
+|RepositoryApiClient|This is the public interface the developer uses to get data from and set value to the Laserfiche
+Repository|RepositoryApiClient|RepositoryApiClientImpl|
+|OAuthInterceptor|It uses [lf-api-client-core-java](https://github.com/Laserfiche/lf-api-client-core-java) to get access
+token from the OAuth service, and converts the token into a Bearer header for each outgoing HTTP
+request.|okhttp3.interceptor|OAuthInterceptor|
+|[Retrofit2](https://square.github.io/retrofit/)/[OkHttp3](https://square.github.io/okhttp/)|We use Swagger Codegen to
```

Comment:
Is this outdated? It mentions Retrofit which we no longer use.

Reply:
The original file is replaced.